packages/httpsnippet-client-api/test/__datasets__/headers/index.ts

```diff
@@ -17,9 +17,13 @@ const mock: SnippetMock = {
         value: 'Bar',
       },
       {
-        name: 'X-Bar',
```

Comment:
Changed this casing because `X-Bar` doesn't exist in the spec for this dataset and how we construct HAR objects in the API Explorer we won't ever translate `x-bar` to `X-Bar` (because `X-Bar` isn't in the spec).